tests/fail2ban.py

```diff
@@ -10,11 +10,11 @@
 
 # parse command line
 
-if len(sys.argv) != 3:
-	print("Usage: tests/fail2ban.py \"ssh user@hostname\" hostname")
+if len(sys.argv) != 4:
+	print("Usage: tests/fail2ban.py \"ssh user@hostname\" hostname owncloud_user")
```

Comment:
Are these changes needed for OC 9 or are they additional improvements? If it is the second I would suggest splitting them up into another PR.